jgrapht-core/src/test/java/org/jgrapht/alg/matching/blossom/v5/Benchmarks.java

```diff
@@ -0,0 +1,137 @@
+package org.jgrapht.alg.matching.blossom.v5;
+
+import org.jgrapht.Graph;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.annotations.State;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static org.jgrapht.alg.matching.blossom.v5.Options.DualUpdateStrategy.MULTIPLE_TREE_CONNECTED_COMPONENTS;
+import static org.jgrapht.alg.matching.blossom.v5.Options.DualUpdateStrategy.MULTIPLE_TREE_FIXED_DELTA;
+import static org.jgrapht.alg.matching.blossom.v5.Options.InitializationType.GREEDY;
+import static org.jgrapht.alg.matching.blossom.v5.Options.InitializationType.NONE;
+
+@Fork(value = 1, warmups = 0)
+@BenchmarkMode(Mode.SampleTime)
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Warmup(iterations = 3, timeUnit = TimeUnit.MINUTES, time = 1)
+@Measurement(iterations = 10, timeUnit = TimeUnit.MINUTES, time = 1)
+public class Benchmarks {
+
+    public static Options[] options = new Options[]{
+            new Options(NONE, MULTIPLE_TREE_CONNECTED_COMPONENTS, true, true), //[0]
+            new Options(NONE, MULTIPLE_TREE_CONNECTED_COMPONENTS, true, false), //[1]
+            new Options(NONE, MULTIPLE_TREE_CONNECTED_COMPONENTS, false, true), //[2]
+            new Options(NONE, MULTIPLE_TREE_CONNECTED_COMPONENTS, false, false), //[3]
+            new Options(NONE, MULTIPLE_TREE_FIXED_DELTA, true, true), //[4]
+            new Options(NONE, MULTIPLE_TREE_FIXED_DELTA, true, false), //[5]
+            new Options(NONE, MULTIPLE_TREE_FIXED_DELTA, false, true), //[6]
+            new Options(NONE, MULTIPLE_TREE_FIXED_DELTA, false, false), //[7]
```

Comment:
In the test sheet (result.pdf) it says that option7 is greedy?

Reply:
I’ve reordered them so that we have first parameter intialization type, then dual strategy, then update duals before, then after. I’ve put a second table in the results for that purpose.